website/docs/02-installation/02-upgrading.mdx

```diff
@@ -54,7 +54,7 @@ please **make sure to create in-game saves** before upgrading.
 2. Copy the 7 folders to your SD card, using the following command (replace `/run/media/mmcblk0p1` with the path to your SD card):
 
    ```sh
-   cp -rf Onion-v4.2.0/* /run/media/mmcblk0p1
+   /bin/cp -rfv Onion-v4.2.0/* /run/media/mmcblk0p1
```

Comment:
This would also work, at least in bash or zsh:
```sh
$(which cp) -rf Onion-v4.2.0/* /run/media/mmcblk0p1
```

I do think I'm more inclined to use the `yes |` method.